app/helpers/admin/authors_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin::AuthorsHelper
+  def author_image_tag author, size
+    if author.thumb_img.attached?
+      image_tag author.thumb_img, size:, class: "img-circle"
+    elsif author.thumb.present?
+      image_tag author.thumb, size:, class: "img-circle"
+    else
+      image_tag "user.png", size:, class: "img-circle"
```

Comment:
```suggestion
      author.thumb_img
    elsif author.thumb.present?
      author.thumb
    else
      "user.png"
```
thẻ image_tag ở bên ngoài html nhé chỗ ảnh thì mới gọi helper này thôi
Em đang bị lặp code ở image_tag với size:, class: "img-circle"